src/test/visual/graphqlMocks.ts

```
@@ -0,0 +1,80 @@
+
+export const successResponse: (route: Route) => void = async (route: Route): Promise<void> => {
+  const request: Request = route.request();
+  let postData: GraphQLRequestPayload;
+  try {
+    postData = await request.postDataJSON();
+  } catch (error) {
+    await route.continue();
+    return;
+  }
+
+  if (postData && postData.query && postData.query.includes('mutation AddUser')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)